pinot-query-planner/src/test/java/org/apache/pinot/query/QueryEnvironmentTestBase.java

```diff
@@ -57,6 +57,8 @@ protected Object[][] provideQueries() {
             + " GROUP BY a.col1, a.col2"},
         new Object[]{"SELECT a.col1, AVG(b.col3) FROM a JOIN b ON a.col1 = b.col2 "
             + " WHERE a.col3 >= 0 AND a.col2 = 'a' AND b.col3 < 0 GROUP BY a.col1"},
+        new Object[]{"SELECT a.col1, COUNT(*) FROM a WHERE a.col3 >= 0 AND a.col2 = 'a' GROUP BY a.col1"
+            + " HAVING COUNT(*) > 10"},
```

Comment:
Suggest adding one test each for each supported comparison operator and may be also include a different aggregation function